### PR TITLE
[chore](cloud) Support starting both meta-service and recycler within single process

### PR DESCRIPTION
**Pull request by gavinchou**
e.g. the following will start  meta-service and recycler within single process. 
```
./bin/start.sh --daemon
```
the log file will be meta_service.INFO*

and it is the same effect as `./bin/start.sh --meta-service --recycler --daemon`

doc PR https://github.com/apache/doris-website/pull/1073

### CODE REVIEW

**Comment by gavinchou** [comment]
run buildall

File: cloud/script/start.sh [hunk 1]
```diff
@@ -131,14 +131,22 @@ fi
...
-    nohup "${bin}" "$@" >>"${DORIS_HOME}/log/${process}.out" 2>&1 &
-    # wait for log flush
-    sleep 1.5
-    tail -n10 "${DORIS_HOME}/log/${process}.out" | grep 'working directory' -B1 -A10
-    echo "please check process log for more details"
-    echo ""
+    # append 10 blank lines to ensure the following tail -n10 works correctly
+    printf "\n\n\n\n\n\n\n\n\n\n" >> "${out_file}"
+    echo "$(date +'%F %T') try to start ${process}" >> "${out_file}"
+    nohup "${bin}" "$@" >> "${out_file}" 2>&1 &
```
Comment (gavinchou) [inline cloud/script/start.sh:139]:
```suggestion
    printf "\n\n\n\n\n\n\n\n\n\n" >>"${out_file}"
    echo "$(date +'%F %T') try to start ${process}" >>"${out_file}"
    nohup "${bin}" "$@" >>"${out_file}" 2>&1 &
```

**Comment by gavinchou** [comment]
run buildall

**Review by airborne12 — DISMISSED** [review]
LGTM

**Review by bobhan1 — APPROVED** [review]

**Review by liaoxin01 — DISMISSED** [review]
LGTM

**Comment by gavinchou** [comment]
run buildall

**Comment by gavinchou** [comment]
run buildall

File: cloud/src/common/config.h [hunk 2]
```diff
@@ -77,9 +77,11 @@ CONF_mInt32(scan_instances_interval_seconds, "60"); // 1min
 CONF_mInt32(check_object_interval_seconds, "43200"); // 12hours
 
 CONF_mInt64(check_recycle_task_interval_seconds, "600"); // 10min
+CONF_mInt64(recycler_sleep_before_scheduling_seconds, "60");
 // log a warning if a recycle task takes longer than this duration
 CONF_mInt64(recycle_task_threshold_seconds, "10800"); // 3h
 
+
```
Comment (gavinchou) [inline cloud/src/common/config.h:84]:
```suggestion
```

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**